image/templates/helm/stackrox-secured-cluster/templates/_init.tpl.htpl

```diff
@@ -66,6 +66,13 @@
 
 {{- include "srox.checkVersionRequirements" $ -}}
 
+{{/* Sanity checks.*/}}
+{{- if not (kindIs "invalid" ._rox.clusterLabels) }}
+  {{- if not (kindIs "map" ._rox.clusterLabels) }}
+    {{ include "srox.fail" (printf "type mismatch: clusterLabels is a %s, but must be a dict" (kindOf ._rox.clusterLabels)) }}
+  {{- end }}
```

Comment:
Could we maybe iterate over entries and check they strings and not numbers or booleans? 🤔